src/Umbraco.Cms.Api.Management/Controllers/Document/ByKeyDocumentController.cs

```diff
@@ -53,7 +71,9 @@ public async Task<IActionResult> ByKey(CancellationToken cancellationToken, Guid
             return DocumentNotFound();
         }
 
-        DocumentResponseModel model = await _documentPresentationFactory.CreateResponseModelAsync(content);
+        ContentScheduleCollection schedule = _contentService.GetContentScheduleByContentId(content.Id);
```

Comment:
It's not optimal to call two services from the controller. I think we should make it a single service call and move the logic to the service layer. Furthermore, when using two services, we should wrap it in a single scope, to avoid funny things that can happen outside of a transaction